pages/index.html

```diff
@@ -4,9 +4,33 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../stylesheets/style.css" rel="stylesheet">
+  <link href="../stylesheets/index.css" rel="stylesheet">
+  <title>Home</title>
 </head>
 <body>
-  
+  <header>
+    <h1>
+      Morgan Buntin
+    </h1>
+    <nav>
+      <a href="./about.html">About Me</a>
+      <a href="./portfolio.html">My Work</a>
```

Comment:
These relative paths will probably need to be updated if index.html is moved to the top level